contracts/modules/LoanOpenings/LoanOpenings.sol

```diff
@@ -89,6 +94,79 @@ contract LoanOpenings is State, LoanOpeningsEvents, VaultController, InterestHan
         );
     }
 
+    function getDefaultLoanParams(
+        address loanToken,
+        address collateralToken,
+        bool isTorqueLoan)
+        public
+        view
+        returns(LoanParams memory loanParamsLocal) 
+    {
+        
+        bytes32 loanParamsId = getPoolLoanParamId(loanToken, collateralToken, isTorqueLoan);
+
+        loanParamsLocal = loanParams[loanParamsId];
+        if (loanParamsLocal.id == 0) {
+            if (isTorqueLoan) {
+                // address loanPool = underlyingToLoanPool[collateralToken];
+                loanParamsLocal.active = true;
+                // loanParamsLocal.owner = loanPool; // ZERO for autogenerated
+                loanParamsLocal.loanToken = loanToken;
+                loanParamsLocal.collateralToken = collateralToken;
+                loanParamsLocal.minInitialMargin = 20 ether;
+                loanParamsLocal.maintenanceMargin = 15 ether;
+                loanParamsLocal.maxLoanTerm = 0;
+                loanParamsLocal.id = getLoanParamId(loanParamsLocal);
+            } else {
+                // we can't use iToken as collateral for marginTrade yet
+            }
+        } else {
+            return loanParamsLocal;
+        }
+    }
+
+    function getPoolLoanParamId(
+        address loanToken,
+        address collateralToken,
+        bool isTorqueLoan)
+        public
+        view
+        returns (bytes32) 
+    {
+        return loanParamsIds[keccak256(abi.encodePacked(
+            loanToken,
+            collateralToken,
+            isTorqueLoan
+        ))];
+    }
+
+    function getLoanParamId(
+        LoanParams memory loanParam)
+        public
+        pure
+        returns (bytes32) 
+    {
+        return keccak256(abi.encode(
+            loanParam.loanToken,
+            loanParam.collateralToken,
+            loanParam.minInitialMargin,
+            loanParam.maintenanceMargin,
+            loanParam.maxLoanTerm
+        ));
+    }
+
+    function upgradeLoanParams(
```

Comment:
prefix with _ on function name since internal. Aids in keeping track of function names.

Reply:
looks like this function is not needed at all